scripts/languages.js

```diff
@@ -0,0 +1,48 @@
+const fs = require('fs')
+
+const LinguistLanguages = require('linguist-languages')
+
+/**
+ *
+ * @param {string} name
+ * @returns
+ */
+const getSupportLanguages = (name) => {
+  const language = LinguistLanguages[name]
+  return [
+    {
```

Comment:
```suggestion
    {
      ...language,
```

This way, we might end up with a few extra fields that are never used, but that doesn't really matter.

Reply:
Hah, right, I was still thinking on the `yaml` format with different keys.